shared/test/test_properties.rb

```diff
@@ -29,6 +29,12 @@ def test_get_set_delete
     set_key_value(key, value_boolean)
     assert_equal value_boolean, get_key_value(key)
 
+    assert JSON.parse(list(@properties, @channel_id)).length == 1, "Owner can list all properties."
+
+    other_properties = Rack::Test::Session.new(Rack::MockSession.new(PropertiesApi, "studio.code.org"))
```

Comment:
Nit: Could use a comment here explaining that you're setting up a session that does not share cookies with the standard test session.